pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +111,82 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    timestamps = pd.DatetimeIndex(MELBOURNE_FL[0])
+    timestamps = timestamps.tz_localize('UTC').tz_convert('Etc/GMT+5')
```

Comment:
This was not tz aware on my machine, it gets converted to a naive date time index in utc, that's why I had to localize as UTC then convert to us/Eastern. Is this a pandas or pytz version issue?